cli/readInputs.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+// This function allows for increased functionality of the Vault helper tool
+// Reads one line of user input at a time
+func readInput() {
```

Comment:
Hmm sorry I know I asked you to name this `readInput()` in the last PR. Now that I see the whole app, could you name it something indicating that it's an interactive console that's an extension of `main()`? ex. `interactiveApp()`.